Tests/SourceKitLSPTests/RangeFormattingTests.swift

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import SKLogging
+import SKTestSupport
+import SourceKitLSP
+import XCTest
+
+final class RangeFormattingTests: XCTestCase {
+  func testSelectComment() async throws {
+    try await SkipUnless.toolchainContainsSwiftFormat()
+    let testClient = try await TestSourceKitLSPClient()
+    let uri = DocumentURI(for: .swift)
+
+    let positions = testClient.openDocument(
+      """
+      func foo() {
+      if let SomeReallyLongVar = Some.More.Stuff(), let a = myfunc() {
+      1️⃣// do stuff2️⃣
+      }
+      }
+      """,
+      uri: uri
+    )
+
+    let response = try await testClient.send(
+      DocumentRangeFormattingRequest(
+        textDocument: TextDocumentIdentifier(uri),
+        range: positions["1️⃣"]..<positions["2️⃣"],
+        options: FormattingOptions(tabSize: 2, insertSpaces: true)
+      )
+    )
+
+    let edits = try XCTUnwrap(response)
```

Comment:
I don’t think you need to unwrap here. You should be able to compare the optional `response` to the array literal straight away.

Reply:
That's how it's done in `FormattingTests` so I think I should keep it that way for consistency?

Reply:
OK, fair enough. Let’s keep the `XCTUnwrap` then.